cbcontainers/state/hardening/objects/enforcer_mutating_webhook.go

```diff
@@ -0,0 +1,202 @@
+package objects
```

Comment:
The contents of the `enforcer` and `validating` appliers is more or less the same since we want to apply the same configurations to both.
Could we create 1 common webhook applier that has most of the common logic and applies it using adapters - and then only have the specifics like `NewEnforcerMutatingWebhookK8sObject` or `MutateHardeningChildK8sObject` left? They would then call the common version for setting the rules, selectors, etc. 

I think we'll avoid maintaining the same config for both webhooks that way and can focus only on the specifics needed for enforce vs validate (if any)

Reply:
added a ticket for that: GRC-1554